bot/pagination.py

```diff
@@ -229,20 +229,35 @@ def event_check(reaction_: discord.Reaction, user_: discord.Member) -> bool:
                 or isinstance(user_, Member) and any(role.id in MODERATION_ROLES for role in user_.roles)
             )
 
-            return (
+            right_reaction = (
+                # Reaction is on this message
+                reaction_.message.id == message.id
+                # Reaction is one of the pagination emotes
+                and str(reaction_.emoji) in PAGINATION_EMOJI
+            )
+
+            whitelisted = (
                 # Conditions for a successful pagination:
                 all((
-                    # Reaction is on this message
-                    reaction_.message.id == message.id,
-                    # Reaction is one of the pagination emotes
-                    str(reaction_.emoji) in PAGINATION_EMOJI,
                     # Reaction was not made by the Bot
                     user_.id != ctx.bot.user.id,
+                    # It was the right reaction and Message
+                    right_reaction,
                     # There were no restrictions
                     no_restrictions
                 ))
             )
 
+            if whitelisted:
+                return True
+
+            elif right_reaction and user_.id != ctx.bot.user.id:
+
+                ctx.bot.loop.create_task(reaction_.message.remove_reaction(reaction_.emoji, user_))
```

Comment:
Use `asyncio.create_task` instead. In fact, it would be even better to use [`scheduling.create_task`](https://github.com/python-discord/bot/blob/main/bot/utils/scheduling.py#L160) since it logs exceptions. I believe that even with `asyncio.create_task`, any exception raised would eventually show up anyway as uncaught (typically when the bot shuts down), so it's worth handling it. In this case, we'd want to suppress the exception raised when the message no longer exists.

Reply:
So pretty much I should change to `scheduling.create_task` and suppress `discord.HttpException` (or just `NotFound`)?